Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/DeployToSqlServer.ps1

```diff
@@ -0,0 +1,131 @@
+﻿Import-Module $env:CURRENT_TASK_ROOTDIR\DeploymentSDK\InvokeRemoteDeployment.ps1
+
+function EscapeSpecialChars
+{
+    param(
+        [string]$str
+    )
+
+    return $str.Replace('`', '``').Replace('"', '`"').Replace('$', '`$')
+}
+
+function IsPublishProfileEmpty
+{
+    param(
+        [string]$publishProfile
+    )
+
+    $publishProfile = $publishProfile.Trim('\', ' ').Trim()
+
+    return ([string]::IsNullOrWhitespace($PublishProfile) -or
```

Comment:
$publishProfile is always empty string or something else but it can't be null. Fixed casing issue